etna/transforms/math/limit.py

```diff
@@ -0,0 +1,122 @@
+from typing import List
+import numpy as np
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.datasets import set_columns_wide
+from etna.transforms.base import ReversibleTransform
+from etna.transforms.utils import match_target_quantiles
+
+
+class LimitTransform(ReversibleTransform):
+    """LimitTransform limits values of some feature between the borders"""
+
+    def __init__(self, in_column: str, lower_bound: float = -1e10, upper_bound: float = 1e10):
```

Comment:
May be it is better to use constants from numpy as defaults [see](https://numpy.org/doc/stable/reference/constants.html)

Reply:
In fact, better to use None as default value
- (None, None) -> x
- (a, None) -> log(x - a)
- (None, b) -> - log(b - x)